### PR TITLE
Provide a shields.io badge

### LINKED ISSUE #1995: Provide a shields.io badge
### Describe the feature you would like

It would be nice to offer a [shields.io](https://shields.io) badge for Foundry, which would contain the logo currently found in the README:

foundry-logo

This would make it easier for devoted users to signal their allegiance to Foundry in their project READMEs.

### PR DESCRIPTION
**Pull request by PriyankMkwna**
I Have made a Foundry shield badge .You can customize the label at left and message at right . In the link "https://img.shields.io/badge/Foundry-Forged_in_Fire......" you can customize the label by replacing "Foundry" and you can customize the message by replacing "Forged_in_Fire". In message for space, we have to provide a underscore ('_'). Also you can customize where to redirect user if they click the badge by replacing "link=https%3A%2F%2Fbook.getfoundry.sh%2F" at the end.

Fixed #1995

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,6 +3,7 @@
...
...Qaxbt46RSIT79u3j0NAQb926RVVVOT4+TqvVyvz8fD0YDC5NTk6ysbHxlCRJ/5KSlAAURyKRTFNTkwAg7t69S5/Px76+Pq7GyMgI9+/fz9HRUQIQO3bsEKOjo38DsJCUJADw+/0BVVW7otHo8ps3b4yvXr3CxMQETCYTTCYTNE0DAOTl5SGXy0FRFOzZswdmsxkVFRXLNTU1xmg0+kNvb+/3AGAcGBiI7969Wwcg6urq+OTJE967d49btmzh9PT0R3WJRIKBQIDBYJBTU1NsaGggAGGz2fTe3t5fAeQZAWwuLi4uP3nypOT1emEwGFBeXo7a2losLCygoaEB/f39MJlMCIVCkCQJBw8ehNVqhcfjQXNzs1RSUiKtX7++DEAZqqqq3KFQiABYUFDAM2fOkCQXFxdJkvfv32dhYSG9Xi+vXbvG2dnZj4oDgQCLioqoKAqHhobodDq/Mc7NzUklJSUIBoOw2WzYtm0blpeXsWbNGkxMTODp06doa2vD4OAgNm7cCIvFApLQdR3nzp3Dzp078fLlSxQVFeHdu3cAgIpHjx69/zBUX5k+MDBAt9vNY8eOsbu7m6lUigcOHKDL5WImkyHJz9TGYrEcALsMIPn69esZTdMIgM+ePUNXVxdu376NsrIyuN1uXLp0CWazGcPDw3C5XFBVFWfPnkVNTQ18Pp+ezWY5MzPzO4DfAABHjhzpJslUKqVdvHiR4+PjbG9vZy6XI0kuLS0xmUxSCEGS9Pv9LC0tpdFoZGVlpSaEoM/nuwIAKx/7q5GRkb9CoZBQVVWcP3+ez58/J0mm02kODg7ywoULjMViTKfTtNvtXLt2LTdt2qTncrnlsbGxLICvSUqfrl5HJBLh1NTUkhBCJ8mFhQX29/dTVVUWFBTwwYMH1HWdly9fpqIoeiKRWJqfn2d1dXWnLMuf7zMAHD16tGd+fn7FZy2bzYrKykodAAFQVVV9cXFRkNTevn3Lubk5trS0XPnfxHE4HN8ODw+nV/yanp6mx+Ohx+P5aIMQgmNjY3/W1tZ+t5rsSwG7+fjx4/76+vrm7du32woLC00AkE6n38fj8ZmHDx/+cuPGjR8BJL8YsCtYdQIMALYqilKvKEo9APuHty+egH8A3GfFDJXmxmMAAAAASUVORK5CYII%3D&link=https%3A%2F%2Fbook.getfoundry.sh%2F)
```
Comment (mattsse) [inline README.md:6]: thanks for the shield,
what do others think about "Forged_in_Fire"?
@DaniPopes @klkvr ?

perhaps we just keep `Foundry`?

Comment (klkvr) [reply README.md:6]: > perhaps we just keep `Foundry`?

yeah, I'd prefer that too

Comment (PriyankMkwna) [reply README.md]: I have made that change.

**Comment by PriyankMkwna** [comment]
I have made changes as you have suggested .

**Review by mattsse — APPROVED** [review]

**Pull request merged**